### PR TITLE
minor edits docs

### PR DESCRIPTION
**Pull request by cantbewong**

### CODE REVIEW

**Comment by akutz** [comment]
Hi @cantbewong 

Thanks! Do you mind closing this and reopening it as a PR for [the RC4 branch](https://github.com/akutz/rexray/tree/release/0.4.0-rc4)? If you're cool with it I am happy to simply grab these changes and merge it myself manually.

**Comment by cantbewong** [comment]
OK you can merge it yourself

**Comment by akutz** [comment]
Hi @cantbewong,

Thanks. Changes look good! I just want to make sure everyone is reviewing the docs in the same place.

**Comment by akutz** [comment]
@cantbewong,

I'm going to merge it via this PR, but I need you to correct something first. It's small. See the line note I'm about to add.

File: .docs/user-guide/application.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ including but not limited the following examples.
         $ docker run -d --volume-driver=rexray -v mongodb:/data/db mongo
 
  * [MySQL](https://hub.docker.com/_/mysql/)
+ 
+         $ docker volume create --driver=rexray --name=mysqldemovolume --opt=size=<sizeInGB>
```
Comment (akutz) [inline .docs/user-guide/application.md]: @cantbewong,

Lines 73-74 have an incongruent amount of leading whitespace characters. Based on the other lines it appears line 73 should be have one less leading space.

**Comment by akutz** [comment]
I can also merge as is and make the edits. I'm going to remove the applications lacking examples anyway. They shouldn't be there.

**Pull request closed without merging**